pkg/daemon/manager/manager.go

```diff
@@ -23,6 +29,17 @@ type Manager interface {
 	EnsureServiceState(winsvc.Service, svc.State) error
 }
 
+// enumServiceStatus implements the ENUM_SERVICE_STATUS type as defined in the Windows API
+type enumServiceStatus struct {
+	ServiceName   *uint16
+	DisplayName   *uint16
+	ServiceStatus windows.SERVICE_STATUS
+}
+
+// enumDependentServicesW is a handle to the EnumDependentServicesW syscall
+// https://learn.microsoft.com/en-us/windows/win32/api/winsvc/nf-winsvc-enumdependentservicesw
+var enumDependentServicesW = windows.NewLazySystemDLL("Advapi32.dll").NewProc("EnumDependentServicesW")
```

Comment:
Does this need to be a global variable? If there are performance implications that made you add this as a global variable, please call that out here.